pkg/nuctl/test/get_test.go

```diff
@@ -45,6 +45,103 @@ func (suite *GetTestSuite) SetupSuite() {
 	})
 }
 
+func unduplicate(array []string) []string {
```

Comment:
This code shouldn't be repeated just so that the test may remain generic...
Here's roughly what I'd do here to keep the test clean:
```go
nuctlArgumentSets := [][]string{
    {functionNames[0], functionNames[1], functionNames[2]},
    {functionNames[0], functionNames[2], functionNames[2]},
    {functionNames[3], functionNames[1]},
    {functionNames[3], functionNames[3]},
}

uniqueFunctionSets := [][]string{
    {functionNames[0], functionNames[1], functionNames[2]},
    {functionNames[0], functionNames[2]},
    {functionNames[3], functionNames[1]},
    {functionNames[3]},
}

// iterate setIndex, argumentSet := range nuctlArgumentSets
    // execute nuctl get with argumentSet
    // scan output buffer and count occurrences of each function name in uniqueFunctionSets[setIndex]
    // verify that each function in uniqueFunctionSets[setIndex] appeared exactly once
```

Note to please document that the below `break` only stops looking for function names in that line and allows scanner.Scan to move on to the next one.